src/Service.Tests/SqlTests/SqlTestHelper.cs

```diff
@@ -354,7 +354,7 @@ public static SqlException CreateSqlException(int number, string message = "")
 
             // Create SqlError object.
             // For details on what the parameters stand for please refer:
-            // https://learn.microsoft.com/en-us/dotnet/api/system.data.sqlclient.sqlerror.number?view=dotnet-plat-ext-6.0#examples
+            // https://learn.microsoft.com/en-us/dotnet/api/system.data.sqlclient.sqlerror.number?view=dotnet-plat-ext-8.0#examples
```

Comment:
for these doc links, is there a non-version specific link, so that next time we need to update - this remains a no-op?

Reply:
if we remove the `?view=dotnet-plat-ext-8.0` it will use the default page which ought to be the latest version

`https://learn.microsoft.com/en-us/dotnet/api/system.data.sqlclient.sqlerror.number#examples`

ill add this in.